tests/cmds/test_legal_hold.py

```diff
@@ -212,6 +285,20 @@ def active_and_inactive_legal_hold_memberships_response(mocker):
     return [_create_py42_response(mocker, ALL_ACTIVE_AND_INACTIVE_CUSTODIANS_RESPONSE)]
 
 
+@pytest.fixture
+def events_response(mocker):
```

Comment:
Is this fixture used anywhere? I am not seeing.
Perhaps this and `EVENTS_RESPONSE` can be removed?

Reply:
Yes, this is no longer needed. See my comment on the item above.